File: woodroid/src/main/java/me/gilo/woodroid/models/filters/OrderNoteFilter.java
package me.gilo.woodroid.models.filters;

import java.util.HashMap;
import java.util.Map;

public class OrderNoteFilter extends  Filter{

    String type;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;

        addFilter("type", type);
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/models/filters/OrderNoteFilter.kt
package me.gilo.woodroid.models.filters

import java.util.HashMap

class OrderNoteFilter : Filter() {

    internal lateinit var type: String

    fun getType(): String {
        return type
    }

    fun setType(type: String) {
        this.type = type

        addFilter("type", type)
    }
}
